gripper:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  right_gripper_controller:
    type: effort_controllers/JointPositionController
    joint: right_gripper_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  left_gripper_controller:
    type: effort_controllers/JointPositionController
    joint: left_gripper_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
